### ISSUE TITLE
Broken on Mac

### ISSUE THREAD

**Issue by 1of13** [OpenDevin/OpenDevin#153]
**Describe the bug**
After following the installation instructions, I can see the "Initializing agent..." message. However, I can't type anything.

**Steps to Reproduce**
1. Follow the installation instructions
2. Follow the instructions to launch OpenDevin

**Expected behavior**
After waiting for initialization, I expect to be able to interact with OpenDevin.

**Actual behavior**
Everything looks fine, but I cannot click or type on the input box. However, I can interact with the right pane.

**Additional context**

<img width="473" alt="SCR-20240325-lzwh" src="https://github.com/OpenDevin/OpenDevin/assets/31781349/c68d0d04-d54a-4134-b8e2-0a55dc891604">

**Comment by geohotstan** [comment]
what does the log from `uvicorn opendevin.server.listen:app --port 3000` look like?

**Comment by w00dbury** [comment]
I have the same issue.

```
$ uvicorn opendevin.server.listen:app --port 3000
INFO:     Started server process [16074]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.1:3000 (Press CTRL+C to quit)
```

**Comment by geohotstan** [comment]
try `ctrl + r` your `http://localhost:3001/`?

**Comment by darklaw00** [comment]
I get this error:
OpenDevin\opendevin\sandbox\sandbox.py", line 51, in __init__
    uid = os.getuid()
AttributeError: module 'os' has no attribute 'getuid'. Did you mean: 'getpid'?

**Comment by rbren** [comment]
@darklaw00 see https://github.com/OpenDevin/OpenDevin/issues/169--a fix has been merged

**Comment by rbren** [comment]
@1of13 the frontend isn't seeing the backend for some reason. Can you open the browser console's network tab, and see what's going on with the `/ws` endpoint?

**Comment by 1of13** [comment]
I tried your latest update. Now, the installation fails after the second step in terminal 2. I ran the command: `npm install` and got the following output:

'''
npm WARN deprecated domexception@4.0.0: Use your platform's native DOMException instead
npm WARN deprecated abab@2.0.6: Use your platform's native atob() and btoa() methods instead

> opendevin-frontend@0.1.0 prepare
> cd .. && husky install frontend/.husky

husky - .git can't be found (see https://typicode.github.io/husky/#/?id=custom-directory)
npm notice
npm notice New minor version of npm available! 10.2.4 -> 10.5.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.5.0
npm notice Run npm install -g npm@10.5.0 to update!
npm notice
npm ERR! code 1
npm ERR! path /.../OpenDevin-main/frontend
npm ERR! command failed
npm ERR! command sh -c cd .. && husky install frontend/.husky

'''
